### DISCUSSION TITLE
How to Use WordCloud in ui.echarts？

### DISCUSSION THREAD

**Discussion by SparkBayes** [zauberzeug/nicegui#1702] [Q&A]
### Question

How to Use WordCloud in ui.echarts

**Comment by falkoschindler** [comment]
Please elaborate: What is WordCloud, how would you normally use it, what have you already tried to use it with NiceGUI, what exactly does not work?

**Reply by SparkBayes** [reply]
I'm glad to have your attention. I would like to introduce “[echarts-wordcloud.min.js](https://github.com/ecomfe/echarts-wordcloud)” into the project for using ui.echarts to draw a word cloud. However, I'm not sure how to proceed. Can you help me with this?

![image](https://github.com/zauberzeug/nicegui/assets/18264797/f36c2c07-d632-41ff-96a2-dfd568e2d0bf)
